collection/file-managers/gather-ftpinfo-information.yml

```diff
@@ -0,0 +1,19 @@
+rule:
+  meta:
+    name: gather ftpinfo information
+    namespace: collection/file-managers
+    author: "@_re_fox"
+    scope: function
+    att&ck:
+      - Credential Access::Credentials from Password Stores [T1555]
+    references:
+      - https://www.ftpinfo.ru/
```

Comment:
This page is not in English. Is there a link to an English version?

Reply:
I searched and have not found any English description of the software.